pwiz_tools/Shared/Common/SystemUtil/ProgressStatus.cs

```diff
@@ -32,6 +32,7 @@ public interface IProgressStatus
         int PercentComplete { get; }
         bool ProgressEqual(IProgressStatus status);
         Exception ErrorException { get; }
+        string WarningMessage { get; }

```

Comment:
Nit, but can you move this up to below Message instead of under ErrorException. I might not have flagged this in the first place, if it had been there as it is in the ProgressStatus class.